pytorch_lightning/trainer/distrib_data_parallel.py

```diff
@@ -328,7 +328,8 @@ def ddp_train(self, gpu_idx, model):
         model = model.configure_ddp(model, device_ids)
 
         # continue training routine
-        self.run_pretrain_routine(model)
+        output = self.run_pretrain_routine(model)
+        return output
```

Comment:
I remember that I was commenting on this in another PR but cannot find it, but basically I was proposing to add this output...